app/react-graphql/hooks/useMutate.tsx

```diff
@@ -37,35 +38,42 @@ export default function useMutate<T extends IJsonObject>(
   if (!sharedConfig || !middleware?.length) {
     throw new Error('sharedConfig and at least one middleware required');
   }
-
-  //Setup the initial mutation Config so it's for sure ready before we get to urql
-  const mutationCfg = useMemo(() => {
-    console.log('memoizing config');
+  const computeConfig = () => {
+    console.log('computing config');
+    console.log('computeConfig -> objectVariables', objectVariables);
     const state = stateFromMutationMiddleware(
       {variables: objectVariables},
       middleware,
       sharedConfig,
     );
-    // console.log('state', state);
+    console.log('state', state.variables);
     return state;
-  }, [sharedConfig, middleware, objectVariables]);
+  };
```

Comment:
To fix the infinite loop I had to get rid of useMemo.  This is used by useState on line 53 so it's setup initially correctly for urql, and then again in an effect on 57 for any variable changes.